lib/puppet/defaults.rb

```diff
@@ -630,12 +644,22 @@ def self.default_vendormoduledir
       * With a Rack Puppet master, you should restart the web server or the
         application server. Passenger lets you touch a `restart.txt` file to
         refresh an application without restarting Apache; see the Passenger docs
-        for details.
-
-      We don't recommend using any value other than `0` or `unlimited`, since
-      most Puppet masters use a pool of Ruby interpreters which all have their
-      own cache timers. When these timers drift out of sync, agents can be served
-      inconsistent catalogs."
+        for details. (Rack Puppet master is deprecated)
+
+      Any value other than `0` or `unlimited` is deprecated, since most Puppet
+      masters use a pool of Ruby interpreters which all have their own cache
```

Comment:
maybe use `Puppet Servers` instead of `Puppet masters` as is done below?

Reply:
updated